maquetado/scss/styles.scss

```diff
@@ -14,11 +14,20 @@ body {
 }
 
 .tech-list {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
   padding: 0 70px;
```

Comment:
I think you can delete this padding and has the same center effect right?

Reply:
Yes, you're right. I missed that when rewriting that part. Already on it.